### PR TITLE
Adapt sub-menus of 'Team' & 'Personal' menus UI

### PR DESCRIPTION
**Pull request by land-bit**
## Description

Please include a summary of the changes and the related issue.

I just adapted the sidebar menus of settings so that it is visible as requested in issue #3013 

## Type of Change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Breaking change
-   [ ] Documentation update

## Checklist

-   [X] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings

## Previous screenshots

Please add here videos or images of previous status

On #3013 issue

## Current screenshots

Please add here videos or images of previous status

Capture d’écran du 2024-09-12 16-19-39

### CODE REVIEW

File: apps/web/lib/settings/left-side-setting-menu.tsx [hunk 1]
```diff
@@ -99,14 +99,9 @@ export const LeftSideSettingMenu = ({ className }: { className?: string }) => {
 								: 'border-l-transparent font-normal dark:text-[#7E7991]'
 						}
 						`}
-						wrapperClassName={`w-full border-t-0 border-r-0 border-b-0 rounded-none
-                font-normal text-[#7e7991] justify-start  pt-[24px] pb-[24px] pl-[24px]
-				border-l-[5px] ${
-					activePage === '/settings/personal'
```
Comment (evereq) [inline apps/web/lib/settings/left-side-setting-menu.tsx:105]: @Cedric921 can you check this PR, why we have this conditional styling and now it's removed?

File: apps/web/lib/settings/left-side-setting-menu.tsx [hunk 2]
```diff
@@ -146,14 +141,9 @@ export const LeftSideSettingMenu = ({ className }: { className?: string }) => {
 								? ' text-[#3826a6] text-primary font-semibold'
 								: ' border-l-transparent font-normal dark:text-[#7E7991]'
 						}`}
-						wrapperClassName={`w-full border-t-0 border-r-0 border-b-0 rounded-none
-						font-normal text-[#7e7991] justify-start text-sm pt-[24px] pb-[24px] pl-[24px]
-	border-l-[5px] ${
-		activePage === '/settings/team'
```
Comment (evereq) [inline apps/web/lib/settings/left-side-setting-menu.tsx:152]: @Cedric921 can you check this PR, why we had this conditional styling and now it's removed?

File: apps/web/lib/settings/left-side-setting-menu.tsx (hunk 1 above)
Comment (land-bit) [reply apps/web/lib/settings/left-side-setting-menu.tsx:105]: @Cedric921 and @evereq , I'd noticed that the style shouldn't be conditioned by the "activePage" variable but with the "open" variable that headlessUI makes available to us in the SidebarAccordian component, so I just moved the code around and didn't delete anything.

File: apps/web/lib/settings/left-side-setting-menu.tsx (hunk 2 above)
Comment (land-bit) [reply apps/web/lib/settings/left-side-setting-menu.tsx:152]: And since these are the same components as the previous one, I just adapted the rest.

**Pull request merged**